frontend/src/metabase-types/api/mocks/table.ts

```diff
@@ -26,10 +25,8 @@ export const createMockSchema = (opts?: Partial<Schema>): Schema => ({
 export const createMockForeignKey = (
   opts?: Partial<ForeignKey>,
 ): ForeignKey => ({
-  origin: createMockField({ id: 1 }),
   origin_id: 1,
-  destination: createMockField({ id: 2 }),
```

Comment:
This caused issues because these fields were overriding other fields with the same id during hydration. Now fields need to be passed explicitly.